openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -167,6 +167,7 @@ class AccountCreationForm(forms.Form):
 
     name = forms.CharField(
         min_length=accounts.NAME_MIN_LENGTH,
+        max_length=accounts.NAME_MAX_LENGTH,
```

Comment:
Instead of relying on system generated string we should add our own error message just like `_NAME_TOO_SHORT_MSG` this will help us to have have translations for them.

Reply:
@DmytroAlipov ^

Reply:
@farhaanbukhsh done 😉